cmd/worker_windows.go

```diff
@@ -78,5 +79,15 @@ func startWorkers(ctx context.Context, workerNum int) error {
 	eg.Go(func() error {
 		return waitWorkers(ctx, pids, handles, args, attr)
 	})
-	return waitSignals(ctx)
+
+	signaled, err := waitSignals(ctx)
+	if signaled {
+		for _, pid := range pids {
+			p, err := os.FindProcess(pid)
+			if err == nil {
+				_ = p.Signal(os.Interrupt)
```

Comment:
https://github.com/golang/go/blob/go1.20.3/src/os/exec.go#L135

Reply:
I used os.Kill here, assuming we will get to the same result :/